docs/reviewer-guide.md

```diff
@@ -19,3 +19,9 @@ Make a few passes over the code you want to review.
 * Reasonable info messages
 * Unit and functional tests to avoid regressions (the core case must be tested)
 * The PR implement what is needed (changes out of the PR scope should belong to a separate one)
+
+## Pull Request structure
+
+* It's preferred that authors rebase on master instead of merging master into their PRs.
+* We merge PRs into our branches.
+* Commits in a PR should make sense: Ask people to squash commits like "Fix reviewer comments", "wip", "addition", ...
```

Comment:
i feel bad asking people to squash commits. it can be really time consuming and doesn't always offer any real value.  

would it be better for us to just say "commit messages should make sense"?

Reply:
Hm, I really think it is fine and pretty normal on github projects that you are asked to squash such commits.

Reply:
ha, I was on PTO and read this PR before the giant ML discussion.

As long as we're balanced about how we ask people to squash things, that's fine.  I just don't want to put unnecessary barriers in place that make people feel like it's cumbersome to contribute to KubeVirt.